src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -108,4 +130,298 @@ public static void report(
         .forEach(viewer -> viewer.sendMessage(prefixedComponent));
     Audience.get(Bukkit.getConsoleSender()).sendMessage(component);
   }
+
+  @Command(
+      aliases = {"staff", "mods", "admins"},
+      desc = "List the online staff members")
+  public void staff(CommandSender sender, Match match) {
+    // List of online staff
+    List<MatchPlayer> onlineStaff =
+        match.getPlayers().stream()
+            .filter(player -> player.getBukkit().hasPermission(Permissions.STAFF))
+            .collect(Collectors.toList());
+    // List of formatted names
+    List<Component> onlineNames =
+        onlineStaff.stream()
+            .map(pl -> pl.getStyledName(NameStyle.FANCY))
+            .collect(Collectors.toList());
+
+    // FORMAT: Online Staff ({count}): {names}
+    Component staffCount =
+        new PersonalizedText(Integer.toString(onlineStaff.size()))
+            .color(onlineStaff.size() >= 1 ? ChatColor.AQUA : ChatColor.RED);
+
+    BaseComponent content =
+        onlineStaff.isEmpty()
+            ? new PersonalizedTranslatable("moderation.staff.empty")
+                .getPersonalizedText()
+                .color(ChatColor.RED)
+                .render()
+            : Components.join(new PersonalizedText(", ").color(ChatColor.GRAY), onlineNames);
+
+    Component staff =
+        new PersonalizedTranslatable("moderation.staff.name", staffCount, content)
+            .getPersonalizedText()
+            .color(ChatColor.GRAY);
+
+    // Send message
+    sender.sendMessage(staff);
+  }
+
+  @Command(
+      aliases = {"mute", "m"},
+      usage = "<player> <reason> -s (silent) -w (warn)",
+      desc = "Mute a player",
+      perms = Permissions.MUTE)
+  public void mute(
+      CommandSender sender,
+      Player target,
+      Match match,
+      @Text String reason,
+      @Switch('s') boolean silent,
+      @Switch('w') boolean warn) {
+    MatchPlayer targetMatchPlayer = match.getPlayer(target);
+
+    // if -w, also warn the player but don't broadcast warning
+    if (warn) {
+      warn(sender, target, match, reason, true);
+    }
+
+    // Call Event for general punishments
+    punish(PunishmentType.MUTE, targetMatchPlayer, sender, reason, silent);
```

Comment:
you should, like the kick command, check if event is cancelled and tell the mod it was cancelled.